Miscellaneous/scf-dictionary

```diff
@@ -0,0 +1,49 @@
+import pickle
+
+from wrap_pyscf import *
+
+scf_guess = {}
+
+filename = "scf_record"
+
+#with open(filename, 'rb') as outfile:
+    #scf_guess = pickle.load(outfile)
+
+if "EthaneA 1Cx2" not in scf_guess:
+    funct_grps = [[0,2,3,4], [1,5,6,7]]
+    scf_guess["EthaneA 1Cx2"] = guess_driver("EthaneA.sdf", funct_grps, [1,1])
+        
+if "PropaneA 1Cx3" not in scf_guess:
+    funct_gprs = [[2,0,1,3], [4,5,6], [7,8,9,10]]
```

Comment:
I updated the coordinates and row indices. I apologize, but I don't recall that we discussed regarding the geometry sdf files and debugging.